src/shared/components/communication-framework/ConversationIcon.scss

```diff
@@ -1,3 +1,13 @@
+@import 'src/styles/common';
+
 .conversationIcon {
+  width: 40px;
+}
+
+.conversationIconWrapper {
+  display: grid;
+  grid-template-columns: 75px 40px;
```

Comment:
`display: grid` -> `display: flex`
`grid-template-columns: 75px 40px;` -> `gap: 16px;`

Not only is this more suitable for the effect you are after, but it will allow you to always apply this class to the wrapper div rather than conditionally, as you are now doing in `ConversationIcon.tsx`